analytics/config.py

```diff
@@ -17,6 +17,7 @@ class DBSettings(PydanticBaseEnvConfig):
      user: str = Field (alias="DB_USER")
      password: Optional[str] = Field(None, alias="DB_PASSWORD")
      ssl_mode: str = Field("require", alias="DB_SSL_MODE")
+     db_schema: str = Field ("app", alias="DB_SCHEMA")
```

Comment:
You would need to set something in local.env as well to override this to be `public` locally.

Reply:
I did that locally but did not commit the changes

Reply:
I'll push it up now